sql/sql_class.h

```diff
@@ -270,11 +270,13 @@ enum enum_binlog_format {
 };
 
 #ifdef WITH_WSREP
-#define WSREP_BINLOG_FORMAT(my_format)                         \
-   ((wsrep_forced_binlog_format != BINLOG_FORMAT_UNSPEC) ?     \
-   wsrep_forced_binlog_format : my_format)
+#define WSREP_BINLOG_FORMAT(thd, my_format)                                 \
+   ((thd->wsrep_forced_binlog_format_override() != BINLOG_FORMAT_UNSPEC) ?  \
+     thd->wsrep_forced_binlog_format_override() :                           \
+     ((wsrep_forced_binlog_format != BINLOG_FORMAT_UNSPEC) ?                \
+       wsrep_forced_binlog_format : my_format))
 #else
-#define WSREP_BINLOG_FORMAT(my_format) my_format
+#define WSREP_BINLOG_FORMAT(thd, my_format) my_format
```

Comment:
There is no WSREP_BINLOG_FORMAT if there is no WITH_WSREP

Reply:
Done